src/entry.js

```diff
@@ -43,23 +43,25 @@ class Entry {
       clock: new Clock(clockId, clockTime),
```

Comment:
I believe, if we do as per the comments above re. the function signature, we can simplify this to:
```
clock: clock || new Clock(identity.id)
```
...and remove lines 35 and 36 (clockId and clockTime).